.github/workflows/sync-tooling.yml

```diff
@@ -0,0 +1,244 @@
+name: Sync Tooling
+
+on:
+  workflow_dispatch:
+    inputs:
+      tracks:
+        description: 'List of repos to rerun the syncing workflow for, formatted as JSON array, e.g. ["exercism/julia"]'
+        required: true
+        default: ''
+  push:
+    branches:
+      - main
+    paths-ignore:
+      - README.md
+      - LICENSE
+
+env:
+  BOT_USERNAME: "exercism-bot"
+  GIT_USERNAME: "Exercism Bot"
+  GIT_EMAIL: "66069679+exercism-bot@users.noreply.github.com"
+
+jobs:
+  fetch-tracks:
+    name: Fetch & fork target repos
+    runs-on: ubuntu-20.04
+
+    outputs:
+      tracks: ${{ steps.fetch-tracks.outputs.result }}
```

Comment:
This still refers to tracks, but this should probably be tooling. Same goes for other mentions in this worklow.